src/Eris/Generator/ContainmentCheckGenerator.php

```diff
@@ -2,7 +2,7 @@
 namespace Eris\Generator;
 use DomainException;
 
-class ContainmentCheck
+class ContainmentCheckGenerator
```

Comment:
IIRC this was an experiment to remove the duplication of `contains()` call inside `Generator::shrink()` implementations. There is a commented out section in `SetGenerator.php`.
The actual fix would be to rename the file, because this is not a class implementing `Generator` and its name is correct as-is.